weaviate/warnings.py

```diff
@@ -227,6 +227,16 @@ def text2vec_huggingface_endpoint_url_and_model_set_together() -> None:
             stacklevel=1,
         )
 
+    @staticmethod
+    def datetime_year_zero(date: str) -> None:
+        warnings.warn(
+            message=f"""Con004: Received a date {date} with year 0. The year 0 does not exist in the Gregorian calendar.
```

Comment:
Extra full stop here after calendar

Reply:
sorry, the comment only appeared after merging. Will do an extra PR where I fix some nits from JP and will include this!